### PR TITLE
Upgraded to PHP-CS-Fixer 2

### PR DESCRIPTION
**Pull request by mickaelandrieu**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
         "lp-digital/github-event-parser": "~0.8"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "1.11.3",
+        "friendsofphp/php-cs-fixer": "v2.0.0",
```
Comment (Quetzacoalt91) [inline composer.json]: Does it allow you to get the future patch releases ?

File: src/AppBundle/PullRequests/BodyParser.php [hunk 2]
```diff
@@ -86,7 +86,7 @@ public function isBackwardCompatible()
         $regex = "/(\|[[:space:]]BC breaks\?[[:space:]]+\|[[:space:]])(.+)\r\n/";
         $backwardCompatible = $this->extractWithRegex($regex);
 
-        return $backwardCompatible == 'yes' ? true : false;
+        return $backwardCompatible === 'yes' ? true : false;
```
Comment (antoin-m) [inline src/AppBundle/PullRequests/BodyParser.php]: Why not `return $backwardCompatible === 'yes';` ?

File: src/AppBundle/PullRequests/BodyParser.php [hunk 3]
```diff
@@ -97,7 +97,7 @@ public function willDeprecateCode()
         $regex = "/(\|[[:space:]]Deprecations\?[[:space:]]+\|[[:space:]])(.+)\r\n/";
         $willDeprecateCode = $this->extractWithRegex($regex);
 
-        return $willDeprecateCode == 'no' ? false : true;
+        return $willDeprecateCode === 'no' ? false : true;
```
Comment (antoin-m) [inline src/AppBundle/PullRequests/BodyParser.php]: Why not `return $willDeprecateCode !== 'no';` ?

**Comment by mickaelandrieu** [comment]
Thanks for review @antoin-m

**Pull request merged**